doc/howtos/building-images.rst

```diff
@@ -171,24 +194,63 @@ Example::
 will create both the raw and the VirtualBox images, both compressed.
 
 
+Base Images
+-----------
 
-Development Images
-------------------
+In your cloned ``ostro-os`` repository folder, ``./meta-ostro/classes/ostro-image.bbclass``
+contains the base definitions for building Ostro OS images. ``./meta-ostro/recipes-image/images/``
+contains some example image recipes.
 
-All images provided by the Ostro Project are intended for
-developers and not directly for production use. To avoid having developers
-accidentally build images for real products that have development
-features enabled, you must make explicit changes in ``local.conf`` to
-enable them.
+The ``ostro-image.bbclass`` can be used in two modes, depending on the ``swupd`` image feature:
 
-Developers building their own images for personal use can follow these
-instructions to replicate the published Ostro OS images. All necessary
-private keys are provided in the ``ostro-os`` repository.
+* swupd active: produces a swupd update stream when building images and in
+  addition defines virtual image recipes which produce image files that are
+  compatible with that update stream.
+* swupd not active: this is the traditional way of building images, where
+  variables directly control what goes into the image.
 
-To do this, before building,  edit the :file:`conf/local.conf` configuration file, 
-find the line
-with ``# require conf/distro/include/ostro-os-development.inc`` and
-uncomment it.
+Developers are encouraged to start building images the traditional way
+by using image recipes like ``ostro-image-noswupd`` where swupd is
+turned off, because:
+
+a) swupd support is still new and may have unexpected problems.
+b) image and swupd bundle creation cause additional overhead
```

Comment:
Can we add a little more on the overhead so people can decide for themselves. For example, is it only overhead on the build systems (compile time and disk space) or is there more to it than that? In other words, what is this 'extra work' that's mentionned in the next line?